.github/workflows/tests.yml

```diff
@@ -291,6 +291,229 @@ jobs:
+          python -m pip install --upgrade 'contourpy>=1.0.1' cycler fonttools \
+              packaging pyparsing python-dateutil setuptools-scm \
+              -r requirements_test.txt sphinx ipython
+          python -m pip install --upgrade pycairo 'cairocffi>=0.8' PyGObject &&
+             python -c 'import gi; gi.require_version("Gtk", "3.0"); from gi.repository import Gtk' &&
+             echo 'PyGObject is available' ||
+             echo 'PyGObject is not available'
+          python -m pip install --upgrade pyqt5  &&
+             python -c 'import PyQt5.QtCore' &&
+             echo 'PyQt5 is available' ||
+             echo 'PyQt5 is not available'
+          python -mpip install --upgrade pyside2 &&
+              python -c 'import PySide2.QtCore' &&
+              echo 'PySide2 is available' ||
+              echo 'PySide2 is not available'
+          yes | python -m pip uninstall wxpython || echo 'wxPython already uninstalled'
+
+      - name: Install matplotlib
+        shell: bash.exe -eo pipefail -o igncr "{0}"
+        env:
+          AUTOCONF: /usr/bin/autoconf-2.69
+          MAKEFLAGS: dw
+        run: |
+          ccache -s
+          git describe
+          # Set flag in a delayed manner to avoid issues with installing other
+          # packages
+          cat <<EOT >> mplsetup.cfg
+          [rc_options]
+          backend=Agg
+
+          [libs]
+          system_freetype = False
+          system_qhull = True
+          EOT
+          cat mplsetup.cfg
+          # All dependencies must have been pre-installed, so that the minver
+          # constraints are held.
+          python -m pip install --no-deps -ve .
+
+      - name: Find DLLs to rebase
+        shell: bash.exe -eo pipefail -o igncr "{0}"
+        run: |
+          find {/usr,/usr/local}/{bin,lib/python3.?/site-packages} /usr/lib/lapack . -name \*.exe -o -name \*.dll -print >files_to_rebase.txt
+
+      - name: Rebase DLL list
+        shell: ash.exe "{0}"
+        run: "rebase --database --filelist=files_to_rebase.txt"
+
+      - name: Check that Matplotlib imports
+        shell: bash.exe -eo pipefail -o igncr "{0}"
+        run: |
+          /usr/bin/python -c "import matplotlib as mpl; import matplotlib.pyplot as plt"
+
+      - name: Set ffmpeg path
+        shell: 'bash.exe -eo pipefail -o igncr "{0}"'
+        run: |
+          oldmplrc=$(python -c "from matplotlib import matplotlib_fname as mplrc_file; print(mplrc_file())")
+          echo "${oldmplrc}"
+          awk '/animation.ffmpeg_path/ { print "animation.ffmpeg_path: /usr/bin/ffmpeg.exe"; next } { print $0; }' "${oldmplrc}" >matplotlibrc
+          mkdir -p ~/.config/matplotlib/
+          cp matplotlibrc ~/.config/matplotlib/matplotlibrc
+          mkdir -p ~/.matplotlib/
+          cp matplotlibrc ~/.matplotlib/matplotlibrc
+
+      - name: Remove default ACLs on /tmp
+        shell: bash.exe -eo pipefail -o igncr "{0}"
+        run: |
+          cd /tmp && setfacl -b .
```

Comment:
Why is this needed?

Reply:
Different attempt to fix the tempfile test; this can go.